pkgs/development/python-modules/django-pwa/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, pytest
+, pytest-django
+, python
+, pythonOlder
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "django-pwa";
+  version = "1.1.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "silviolleite";
+    repo = "django-pwa";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-tP1+Jm9hdvN/ZliuVHN8tqy24/tOK1LUUiJv1xUqRrY=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    django
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+    ${python.interpreter} runtests.py
+    runHook postCheck
+  '';
+
+  meta = with lib; {
+    description = "A Django app to include a manifest.json and Service Worker instance to enable progressive web app behavoir";
+    homepage = "https://github.com/jazzband/django-pwa/";
```

```suggestion
    homepage = "https://github.com/silviolleite/django-pwa";
```
This is a dead link.